__tests__/Unit/Components/Tasks/Card.test.tsx

```diff
@@ -500,4 +500,84 @@ describe('Task card', () => {
 
         expect(updateTaskSpy).toBeCalled();
     });
+
+    it('renders "Not started" if status is AVAILABLE', () => {
+        const DEFAULT_PROPS = {
+            content: {
+                id: 'firestoreDocumentId123',
+                lossRate: {
+                    dinero: 10,
+                    neelam: 5,
+                },
+                links: ['https://realdevsquad.com/learn-site'],
+                completionAward: {
+                    dinero: 110,
+                    neelam: 10,
+                },
+                dependsOn: [],
+                assignee: 'ankur',
+                startedOn: '1618790400',
+                isNoteworthy: true,
+                title: 'test 1 for drag and drop',
+                purpose: 'string',
+                percentCompleted: 0,
+                endsOn: 1618790400,
+                status: TASK_STATUS.AVAILABLE,
+                featureUrl: 'string',
+                type: 'feature',
+                createdBy: 'ankush',
+            },
+            shouldEdit: true,
```

Comment:
Please move these to fixtures.

Reply:
There are no fixtures created for anything throughout the site, the props are passed like this. Should I create a ticket to fix all that?

Reply:
I can't find the fixtures folder in status site. Can you point me to it?

Reply:
Also, seen at a lot of places that the props are used like this only

Reply:
Fixed it

Reply:
@Maheima  by fixtures I meant the mock data folder.